variables.tf

```
@@ -69,7 +69,7 @@ variable "private_subnet_tags" {
 }
 
 variable "enable_create_defaults" {
-  description = "Disable managing the default resources."
+  description = "Add tags to the default resources."
```

This description doesn't say that much to me.
```suggestion
  description = "Replaces AWS default network ACL, security group and routing table with module resources"
```